services/containerinstance/mgmt/2017-10-01-preview/containerinstance/models.go

```diff
@@ -0,0 +1,445 @@
+	// OsType - The operating system type required by the containers in the container group. Possible values include: 'Windows', 'Linux'
+	OsType OperatingSystemTypes `json:"osType,omitempty"`
+	// Volumes - The list of volumes that can be mounted by containers in this container group.
+	Volumes *[]Volume `json:"volumes,omitempty"`
+	// InstanceView - The instance view of the container group. Only valid in response.
+	InstanceView *ContainerGroupPropertiesInstanceView `json:"instanceView,omitempty"`
+}
+
+// ContainerGroupPropertiesInstanceView the instance view of the container group. Only valid in response.
+type ContainerGroupPropertiesInstanceView struct {
+	// Events - The events of this container group.
+	Events *[]Event `json:"events,omitempty"`
+	// State - The state of the container group. Only valid in response.
+	State *string `json:"state,omitempty"`
+}
+
+// ContainerPort the port exposed on the container instance.
+type ContainerPort struct {
+	// Protocol - The protocol associated with the port. Possible values include: 'ContainerNetworkProtocolTCP', 'ContainerNetworkProtocolUDP'
+	Protocol ContainerNetworkProtocol `json:"protocol,omitempty"`
+	// Port - The port number exposed within the container group.
+	Port *int32 `json:"port,omitempty"`
+}
+
+// ContainerProperties the container instance properties.
+type ContainerProperties struct {
+	// Image - The name of the image used to create the container instance.
+	Image *string `json:"image,omitempty"`
+	// Command - The commands to execute within the container instance in exec form.
+	Command *[]string `json:"command,omitempty"`
+	// Ports - The exposed ports on the container instance.
+	Ports *[]ContainerPort `json:"ports,omitempty"`
+	// EnvironmentVariables - The environment variables to set in the container instance.
+	EnvironmentVariables *[]EnvironmentVariable `json:"environmentVariables,omitempty"`
+	// InstanceView - The instance view of the container instance. Only valid in response.
+	InstanceView *ContainerPropertiesInstanceView `json:"instanceView,omitempty"`
+	// Resources - The resource requirements of the container instance.
+	Resources *ResourceRequirements `json:"resources,omitempty"`
+	// VolumeMounts - The volume mounts available to the container instance.
+	VolumeMounts *[]VolumeMount `json:"volumeMounts,omitempty"`
+}
+
+// ContainerPropertiesInstanceView the instance view of the container instance. Only valid in response.
+type ContainerPropertiesInstanceView struct {
+	// RestartCount - The number of times that the container instance has been restarted.
+	RestartCount *int32 `json:"restartCount,omitempty"`
+	// CurrentState - Current container instance state.
+	CurrentState *ContainerState `json:"currentState,omitempty"`
+	// PreviousState - Previous container instance state.
+	PreviousState *ContainerState `json:"previousState,omitempty"`
+	// Events - The events of the container instance.
+	Events *[]Event `json:"events,omitempty"`
```

Comment:
Hello breaking change

Reply:
This is in a completely new API Version, it's not a breaking change. :)

Reply:
Brainfart